### ISSUE TITLE
sacCer3 genome for diffReps

### ISSUE THREAD

**Issue by mayre** [shenlab-sinai/region_analysis#30]
Hello,

I would like to try out diffReps on some ChIP-seq data that I have recently analyzed by MACS2. Could you please add the relevant genome files for this? sacCer3 is the most updated one.

Thank you!
Mayuri

**Comment by lishen** [comment]
Ningyi, have you added this genome already?

Li


On Tue, Feb 17, 2015 at 6:52 PM, mayre <notifications@github.com> wrote:

> Hello,
>
> I would like to try out diffReps on some ChIP-seq data that I have
> recently analyzed by MACS2. Could you please add the relevant genome files
> for this? sacCer3 is the most updated one.
>
> Thank you!
> Mayuri
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shenlab-sinai/region_analysis/issues/30>.
>



-- 
Li Shen, Ph.D. Computer Science
Assistant Professor
Department of Neuroscience
Icahn School of Medicine at Mount Sinai
New York, NY 10029
#################
ngs.plot - the coolest NGS plotting tool: https://code.google.com/p/ngsplot/
Want ChIP-seq differential analysis? Use:
https://code.google.com/p/diffreps/

**Comment by ny-shao** [comment]
No, sacCer3 is not added. sacCer3 is not included in UCSC Refseq database, so I could not add it in `region_analysis`.

**Comment by mayre** [comment]
Hello Ning-Yi,

You are correct that there is no Refseq database, however, according to this question on the UCSC Genome Browser forum, the dump can be downloaded from this (modified) link:

http://hgdownload.soe.ucsc.edu/goldenPath/sacCer3/database/

The links from the forum seem to be broken?...

Please let me know if these files will work. Thank you!

Best,
Mayuri
